exercises/etl/canonical-data.json

```diff
@@ -0,0 +1,62 @@
+{
+  "transform": {
+    "description": "transforms the input map from a map of value: [ keys ] to a map of key: value for each of the keys",
+    "cases": [
+      {
+        "description": "transforms one value",
+        "input": {
+          "1": ["A"]
+        },
+        "expected": {
+          "a" : 1
+        }
+      },
+      {
+        "description": "tramsforms more values",
```

Comment:
i'll fix the typos.  interestingly, the test descriptions are thin but consistently thin across the existing suites.  I can beef them up.  I was mimicking what i'd seen in the existing...